tests/test_find_dates.py

```diff
@@ -34,5 +34,7 @@
     #('11. 12. 2014, 08:45:39', datetime(2014, 11, 12, 8, 45, 39)),
 ])
 def test_find_date_strings(input_text, expected_date):
+    return_date = None
     for return_date in datefinder.find_dates(input_text):
         assert return_date == expected_date
+    assert ((return_date is not None) or (expected_date is None))
```

Comment:
@lindauer: looks good and nice catch

The one test that is failing right now can be commented out. Can you please comment out the test on line 15 of this [file](https://github.com/akoumjian/datefinder/pull/11/files#diff-5fc3b1451e8ffb0b1da4202e415b0884R15) before we merge the PR?

Thanks